index.html

```diff
@@ -633,7 +633,7 @@ <h2>
       <dfn>array</dfn>, and <dfn>string</dfn> as defined in [[[ECMA-404]]]. <dfn data>hexcolor</dfn> is a [=string=] expressing RGB color values [[css-color-3]] in hexadecimal notation: ‘#’ immediately followed by six hexadecimal characters (#rrggbb).
     </p>
     <section id='webappmanifest-members'>
-      <h2>Members inherited from the {{WebAppManifest}} dictionary</h2>      
+      <h3 id="x2-1-members-inherited-from-the-webappmanifest-dictionary">Members inherited from the <a data-link-type="idl" data-type="dictionary" href="https://www.w3.org/TR/appmanifest/#dom-webappmanifest"><code>WebAppManifest</code></a> dictionary<a class="self-link" aria-label="§" href="#webappmanifest-members"></a></h3>          
```

Comment:
`WebAppManifest` doesn’t exist anymore, we should probably use https://www.w3.org/TR/appmanifest/#dfn-application-manifest

BTW, the first `h*` child element in a `section` is the section's title. ReSpec will take any element in the h1-h6 range and renumber it to match the section's nesting depth correctly. It is a common convention (though by no means a requirement) to use `h2` for all sections.

Reply:
Removed the link in https://github.com/w3c/miniapp-manifest/commit/fe1d3f713594aad86aebbb108463afa5f08926a8